### PR TITLE
update to readme

### PR DESCRIPTION
**Pull request by dorahermes**
update to readme and added quick stats

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+Repository to publicly share code to analyze/visualize CCEP data across development in BIDS
 
 
 # Dependencies
   Fieldtrip: http://www.fieldtriptoolbox.org/
+
+
+# Acknowledgements
+The project is funded by the National Institute Of Mental Health of the National Institutes of Health under Award Number R01MH122258 to Dora Hermes (Mayo Clinic).
+
```
Comment (dvanblooijs) [inline README.md:11]: Wil je hier ook Dutch Epilepsy foundation (NEF 17-07) aan toe voegen, want daar ben ik de afgelopen jaren van betaald.

**Review by dvanblooijs — CHANGES_REQUESTED** [review]
Hi, could you add my grant for the past four years as well?

**Comment by dorahermes** [comment]
Added the info, but please edit if it needs to be listed differently

**Comment by dorahermes** [comment]
@dvanblooijs please check

**Review by dvanblooijs — APPROVED** [review]

**Pull request merged**